### ISSUE TITLE
Make it work on Nitrous

### ISSUE THREAD

**Issue by spike01** [makersacademy/unixoid-challenge#23]
On a fresh Nitrous provision, running the gem out of the box gives an 'empty ident name not allowed' error  

Logs:  

```sh
action@forceful-tank-73-176890:~$ unixoid-challenge                                                                                                                           
Running the Unixoid Challenge Assessor ...                                                                                                                                    
Please enter your github user name:                                                                                                                                           
spike01                                                                                                                                                                       
Enter host password for user 'spike01':                                                                                                                                       
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                                                                               
                                 Dload  Upload   Total   Spent    Left  Speed                                                                                                 
100   122  100    93  100    29    189     59 --:--:-- --:--:-- --:--:--   250                                                                                                
                                                                                                                                                                              
*** Please tell me who you are.                                                                                                                                               
                                                                                                                                                                              
Run                                                                                                                                                                           
                                                                                                                                                                              
  git config --global user.email "you@example.com"                                                                                                                            
  git config --global user.name "Your Name"                                                                                                                                   
                                                                                                                                                                              
to set your account's default identity.                                                                                                                                       
Omit --global to set the identity only in this repository.                                                                                                                    
                                                                                                                                                                              
fatal: empty ident name (for <action@forceful-tank-73-176890.(none)>) not allowed                                                                                             
error: src refspec master does not match any.                                                                                                                                 
error: failed to push some refs to 'git@github.com:spike01/unixoid_submission.git'                                                                                            
Completed Submission!              
```

**Comment by prufrock123** [comment]
@spike01 Why did you close this issue? What solution did you find?

**Comment by spike01** [comment]
We were failing to check for and set the identity name before pushing to Github - this was resolved in d9fbd102
